std/uni.d

```diff
@@ -7560,7 +7560,7 @@ public:
         return r.length == 0;
     }
 
-    this(this) pure @nogc nothrow
+    this(this) pure @nogc nothrow @trusted
```

Comment:
This should use stdx.allocator.makeArray - it's `@safe`

```d
import std.experimental.allocator.mallocator : Mallocator;
import std.experimental.allocator : makeArray;

auto p = Mallocator.instance.makeArray!ubyte(20);
```

https://run.dlang.io/is/iZGDzT

Reply:
Can't use it here because it's not `pure`.

Reply:
```
pragma(msg, "std.uni.d:Grapheme.this(this):", __LINE__, " ", typeof(&Grapheme.__postblit));
```